api/v1alpha1/scaledobject_types.go

```diff
@@ -12,6 +12,8 @@ import (
 // +kubebuilder:printcolumn:name="ScaleTargetKind",type="string",JSONPath=".status.scaleTargetKind"
 // +kubebuilder:printcolumn:name="ScaleTargetName",type="string",JSONPath=".spec.scaleTargetRef.name"
 // +kubebuilder:printcolumn:name="Triggers",type="string",JSONPath=".spec.triggers[*].type"
+// +kubebuilder:printcolumn:name="Min",type="integer",JSONPath=".spec.minReplicaCount"
+// +kubebuilder:printcolumn:name="Max",type="integer",JSONPath=".spec.maxReplicaCount"
```

Comment:
just a minor nit, could you please move `Min/Max` before `Triggers`?  Feels better for me 🤷‍♂️ 😄

Reply:
alright sure! should i add scaledJob one in a different PR or it would be good to add in the same?
@zroubalik

Reply:
The same PR is okay, thanks a lot!

Reply:
@Ritikaa96 and update Changelog please :)